pkg/crc/cluster/cluster.go

```diff
@@ -29,7 +29,7 @@ func WaitForSSH(sshRunner *ssh.Runner) error {
 		return nil
 	}
 
-	return errors.RetryAfter(60*time.Second, checkSSHConnectivity, time.Second)
+	return errors.RetryAfter(300*time.Second, checkSSHConnectivity, time.Second)
```

Comment:
from 1 minute to 5?! what is the justification here?

Reply:
Drivers are currently responsible of part of the wait: Start returns once the IP is available. This is not possible to do that with vsock. The startup polling is only here and therefore need to be extended a little bit.

Reply:
'a little bit.' ?

would another optimization be possibile in future? in this case instead of `GetIP` this would need to become a `Dial` and see if the socket responds or a `ping` to see if the other endpoint responds correctly.

Reply:
300 is by security. libvirt driver wait time is 3 * 60sec, hyperkit 2 * 60sec, hyperv driver is infinite (https://github.com/code-ready/machine/blob/master/drivers/hyperv/hyperv.go#L314 - lol) !